lib/registration.pm

```diff
@@ -720,7 +720,14 @@ sub registration_bootloader_params {
 }
 
 sub yast_scc_registration {
-    my $module_name = y2_module_consoletest::yast2_console_exec(yast2_module => 'scc');
+    # for leap to sle migration, we need to install yast2-registration
+    # before running yast2 registration module.
+    my $client_module = 'scc';
+    if (is_leap_migration) {
```

Comment:
Not always need to install yast2-registration even it is for leap migration, such as zypper migration for leap to sles.